src/status_im/hardwallet/core.cljs

```diff
@@ -1752,7 +1753,7 @@
                 :address              address
                 :publicKey            public-key
                 :keycard-instance-uid instance-uid
-                :keycard-key-uid      key-uid
+                :keyUid               (ethereum/normalized-hex key-uid)
```

Comment:
why camelCase?

Reply:
It comes that way from go side, I didn't want to mix cases